specification/personal-demographics.yaml

```diff
@@ -1090,10 +1127,54 @@ paths:
         When the end date for a `temp` or `billing` address passes, local systems should use the patient’s `home` address. It will be extremely rare that no home address is present on a patient record.
 
         There are a number of business rules that should be taken into account:
-        * cannot add more than one `home` address.
-        * `work` address `use` type is a valid response but cannot be added or replaced as it is a legacy value. An address with the `work` type can be removed though.
+        * cannot add more than one of each address use types; `home`, `temp` or `billing`
+        * `work` address `use` type is a valid response but cannot be added or replaced as it is a legacy value. An address with the `work` type can be removed though
         * any `temp` address must have both a `business effective from` and a `business effective to` date. The provision of a business effective to date has particular importance in order to avoid temporary addresses that are no longer relevant to the patient still being held as current data available to any system retrieving the patient record. A suggested default where no actual business effective to is known is 30 days later than the business effective from, up to a maximum of 3 months.
         * any `billing` address must have both a `business effective from` and a `business effective to` date. The provision of a business effective to date has particular importance in order to avoid correspondence addresses that are no longer relevant to the patient still being held as current data available to any system retrieving the patient record. A suggested default where no actual business effective to is known is 30 days later than the business effective from, up to a maximum of 12 months.
+        * the date period is optional; where present they must be valid dates and the `end` date cannot be before the `start` date
+        * the business effective `start` date is optional, however if provided cannot be a future date. If it is not provided it will default to the date of update
```

Comment:
terminology mixed again here business effective is used but in the spec it is not in a field named as that.

Reply:
Changed